##Goal definition


#true iff left is target
bool left_lane

---
##Define result
bool changed_lane
---
#Define a feedback message
